.github/workflows/release.yml

```diff
@@ -80,9 +80,10 @@ jobs:
       - name: Prepare release
         id: prepare-release
         shell: bash
+        env:
+          RELEASE_VERSION: ${{ github.event.pull_request.title }}
```

Comment:
this cant be right. That would set the release version to the title of the PR that triggered the workflow.
However that workflow is not triggered by a pull request.

Reply:
my bad, this is old clipboard content that has been pasted 🤦

Reply:
```suggestion
        env:
          RELEASE_VERSION: ${{ github.event.inputs.version }}
```